server/pulp/server/managers/consumer/agent.py

```diff
@@ -404,6 +418,46 @@ def _unbindings(bindings):
             agent_bindings.append(agent_binding)
         return agent_bindings
 
+    @staticmethod
+    def delete_queue(url, name, consumer_id):
+        """
+        Delete the agent queue.
+        :param url: The broker URL.
```

Comment:
I usually put a whitespace line between the description and the parameters for readability.